api/src/middlewares/services/UserService/__tests__/getUserTemplate.test.ts

```diff
@@ -12,13 +12,21 @@ jest.mock("../../utils/additionalContexts.json", () => {
 });
 
 test("Returns postal template if the form has the metadata value postal: true", () => {
-  expect(getUserTemplate("Brazil", true)).toBe("userPostalConfirmation");
+  expect(getUserTemplate("Brazil", true, "cni")).toBe("userPostalConfirmation");
```

Comment:
do these now need to be updated to reflect the parameter order change?

Reply:
just done